src/mon/Monitor.h

```diff
@@ -637,7 +637,7 @@ class Monitor : public Dispatcher,
   /**
    * Vector holding the Services serviced by this Monitor.
    */
-  std::vector<std::unique_ptr<PaxosService>> paxos_service;
```

Comment:
could you `#include <array>` to ensure that the header is self-contained?

Reply:
Done